src/Package/Impl/ProjectSystem/RProjectLoadHooks.cs

```diff
@@ -83,14 +95,21 @@ internal sealed class RProjectLoadHooks {
         }
 
         [AppliesTo("RTools")]
+#if VS14
         [UnconfiguredProjectAutoLoad2(completeBy: UnconfiguredProjectLoadCheckpoint.CapabilitiesEstablished)]
+#else
```

Comment:
I hope new code is going to be same going forward and not just VS15, but on the other hand, who knows